scripts/release.js

```diff
@@ -130,7 +114,7 @@ prompt([
         process.exit(1);
       }
 
-      const newVersionBranch = `v${newVersion}`;
+      const newVersionBranch = `release/v${newVersion}`;
```

Comment:
Scoping this branch because the default behaviour of `npm version` creates a git tag at `vx.x.x`